letsdrill.rb

```diff
@@ -1,13 +1,20 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+ case 
+ when integer < 60
+  grade = "F"
+when integer < 70
+  grade = "D"
+when integer < 80
+  grade = "C"
+when integer < 90
+  grade = "B"
+else
+   grade = "A"
```

Comment:
The `grade` local variable isn't used after it is assigned a value, so you don't need it.